docs/reference/pc_channel.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Package Control: channel.json"
+---
+
+<!-- Original: https://github.com/wbond/package_control/blob/master/example-channel.json -->
+
+
+# Examples for the channel.json
+
+The latest version of the default channel.json
+can be found in the crawler release assets:  
+https://github.com/packagecontrol/thecrawl/releases/tag/the-channel
+
+
+## Properties
+
+- $schema
+- schema_version
+- repositories
+- packages_cache
+- libraries_cache
+
+"$schema" is used by LSP and LSP-json to provide linting and completions.
+"schema_version" is currently at 4.0.0.
```

Comment:
```suggestion
`$schema` is used by LSP and LSP-json to provide linting and completions.
`schema_version` is currently at 4.0.0.
```

Also link LSP, LSP-JSON, and the 4.0.0 release of `schema_version`?

Reply:
Schemas are currently not hosted anywhere. They are bundled in Package Control's [sublime-package.json](https://github.com/wbond/package_control/blob/master/sublime-package.json).

Ideally schemas would be hosten nearby/next to default channel, maybe.

Haven't published standalone schemas so far, though.